cluster/kubevirtci.sh

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export KUBEVIRT_PROVIDER='k8s-1.17'
+export KUBEVIRT_PROVIDER='k8s-1.18'
```

Comment:
can you please split the PR where the generated/vendor files are in a separate commit?